fs/move.ts

```diff
@@ -30,6 +31,14 @@ export async function move(
     );
   }
 
+  const srcAbsolute = src instanceof URL ? src.pathname : resolve(src)
+  const destAbsolute = dest instanceof URL ? dest.pathname : resolve(dest)
+
+  if (srcAbsolute === destAbsolute) {
+    await Deno.rename(src, dest)
```

Comment:
Is this rename meaningful?

Reply:
I don't feel like it is, since the result will be the same. I put the rename there just in case it would be overriding some behavior during rename.